src/applications/my-education-benefits/components/DuplicateContactInfoModal.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { VaModal } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { acknowledgeDuplicate, toggleModal } from '../actions';
+
+function DuplicateContactInfoModal(props) {
+  useEffect(
+    () => {
+      const allPotentialDuplicates = props?.duplicateEmail?.concat(
+        props?.duplicatePhone,
+      );
+      const filteredPotentialDuplicates = allPotentialDuplicates?.filter(
+        entry => entry?.isDupe === 'true' && entry.acknowledged === undefined,
+      );
+
+      if (filteredPotentialDuplicates?.length === 0) {
+        props.toggleModal(false);
+      } else if (filteredPotentialDuplicates?.length > 0) {
+        props.toggleModal(true);
+      }
```

Comment:
To simplify this a little bit you could use `Array.prototype.some`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

For example:
```
const hasDupe = allPotentialDuplicates?.filter(
  entry => entry?.isDupe === 'true' && !entry.acknowledged,
);
props.toggleModal(!hasDupe);
```